internal/commands/build.go

```diff
@@ -160,9 +159,6 @@ func buildCommandFlags(cmd *cobra.Command, buildFlags *BuildFlags, cfg config.Co
 	cmd.Flags().StringArrayVar(&buildFlags.Volumes, "volume", nil, "Mount host volume into the build container, in the form '<host path>:<target path>[:<mode>]'."+multiValueHelp("volume"))
 	cmd.Flags().StringVarP(&buildFlags.DefaultProcessType, "default-process", "D", "", `Set the default process type. (default "web")`)
 	cmd.Flags().StringVar(&buildFlags.Policy, "pull-policy", "", `Pull policy to use. Accepted values are always, never, and if-not-present. (default "always")`)
-	// TODO: Remove --no-pull flag after v0.13.0 released. See https://github.com/buildpacks/pack/issues/775
-	cmd.Flags().BoolVar(&buildFlags.NoPull, "no-pull", false, "Skip pulling builder and run images before use")
-	cmd.Flags().MarkHidden("no-pull")
 }
 
 func validateBuildFlags(flags *BuildFlags, cfg config.Config, packClient PackClient, logger logging.Logger) error {
```

Comment:
Can we remove the `logger` from this func now, given that it looks like it's not used anymore?

Reply:
The `logger` is still using in this function as the parameter for the `suggestSettingBuilder` func.
https://github.com/buildpacks/pack/blob/7268c89ac5a6bbc6f843f1fdddc0ae598ccf54bd/internal/commands/build.go#L168-L172